main.py

```diff
@@ -1,39 +1,91 @@
 #!/usr/bin/env python
 """Implements math functions without using operators except for '+' and '-' """
+import sys
 
 __author__ = "???"
 
 
 def add(x, y):
-    """Add two integers. Handles negative values."""
-    # your code here
-    return
+    return x + y
 
 
 def multiply(x, y):
     """Multiply x with y. Handles negative values of x or y."""
-    # your code here
-    return
+    if(y == 0):
+        return 0
+
+    # Add x one by one
+    if(y > 0):
```

Comment:
This is awesome, b/c a negative + a negative ... It took me about 5 minutes to remember that. Very cool.